apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/service/SecureStorageServiceImpl.java

```diff
@@ -162,4 +165,22 @@ private SecureStorage collectAllDataToTempStore() throws AplException.ExecutiveP
 
         return secureStorage;
     }
+
+    @Override
+    public boolean flushAccountKeys(Long accountID, String passPhrase) {
+        
+        Set<Entry<Long, String>> setOfEntries = store.entrySet();
```

Comment:
store is a key/value storage, so you can check/get value using key  (store.containsKey( ) / store.get( ))

Reply:
Is the way it's done right now incorrect?

Reply:
done